.azure/templates/jobs/build/deploy_strimzi_java.yaml

```diff
@@ -23,6 +23,14 @@ jobs:
         parameters:
           JDK_VERSION: $(jdk_version)
 
+      # Change release version in pom file
+      - bash: "make release_maven"
+        displayName: "Change release version in pom files"
+        env:
+          RELEASE_VERSION: '${{ parameters.releaseVersion }}'
+        # We want to change it only in case of release (RC or GA), not for SNAPSHOTs
+        condition: eq(variables['build.sourceBranch'], 'refs/heads/release-')
```

Comment:
I'm still a bit confused why is this needed. As you said before, `make release` does this, or? Also, You probably do not want to run this on `main` branch. So if it is needed, it should be decoupled from this job?

Reply:
This is running in different job, where the Strimzi repository for the branch is newly checked-out. That means even if you run it somewhere else, you will have to store the whole repository (and the changes) and then download them here and used them.
I hit this when I was testing the changes. Even when the `make release` was executed in the previous step, the changes were not present here.

> Also, You probably do not want to run this on main branch. So if it is needed, it should be decoupled from this job?
Yes, I have the condition there which will run this only in case of the release branch.

Reply:
I think this probably deserves more thinking and consideration.

Reply:
WDYM? You don't like the idea of having the version set there? Intead of doing this, I can add just `make pushnexus` to the release artifacts job. Or you want to have it completely separate?

Reply:
Yes, I can do that, in that case I wouldn't need to change the deploy to maven job, but I will just add the `make pushtonexus` into the release-artifacts job.